duckduckgo/iOS
# sync delete account

Task/Issue URL: https://app.asana.com/0/0/1204383167189822/f
Tech Design URL:
CC:

**Description**:

Add turn off and delete data

**Steps to test this PR**:
1. Turn on sync and add multiple devices
2. Then choose Turn off and Delete Data
3. All devices should end up turned off



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
